src/main/scala/de/johoop/jacoco4sbt/JacocoPlugin.scala

```diff
@@ -16,9 +16,8 @@ import Keys._
 import java.io.File
 
 import de.johoop.jacoco4sbt.build.BuildInfo
-import inc.Locate
 
-object JacocoPlugin extends Plugin {
+object JacocoPlugin extends AutoPlugin {

```

Comment:
AutoPlugin should declare `override def requires = sbt.plugins.JvmPlugin` if it uses built-in settings and tasks.